### PR TITLE
Force optional

### LINKED ISSUE #11: [Improvement] Add --force parameter to enforce update towards DNS server
This is useful when switching DNS provider.
When A-record is still resolving from the old one with current IP (or it's cached) it's impossible to enforce update of A-record on new DNS provider to assure no-downtime switch of nameservers.

### PR DESCRIPTION
**Pull request by andreeadima**
Add user interaction on domain setup alerting user about the possibility of overriding existing A records (as in applying the DC template with force=True), as per user issue #12.

Also fixes #11.

### CODE REVIEW

**Comment by andreeadima** [comment]
Added @pawel-kow as an owner on the PyPi project, in order to allow Pawel to publish new releases of the https://pypi.org/project/domain-connect-dyndns/.

File: setup.py [hunk 1]
```diff
@@ -5,31 +5,31 @@
...
+    name='domain-connect-dyndns',
+    version='0.0.4',
+    description='Python client library for Dynamic DNS using Domain Connect',
+    license='MIT',
     long_description=long_description,
     long_description_content_type="text/markdown",
-    author = 'Andreea Dima',
-    author_email = 'andreea.dima@1and1.ro',
+    author='Andreea Dima',
+    author_email='andreea.dima@1and1.ro',
```
Comment (pawel-kow) [inline setup.py]: do you want to update the Email maybe?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-language: python
```
Comment (pawel-kow) [inline .travis.yml]: Why removing travis CI ?

File: dyndns/domain_setup.py [hunk 3]
```diff
@@ -61,6 +64,8 @@ def main(domain, settings='settings.txt'):
         api_url=config.urlAPI
     ))
 
+    force = input("Should we override any existing A records in the DNS zone (y/n)?") == 'y'
```
Comment (pawel-kow) [inline dyndns/domain_setup.py]: I think asking for it every time is not the best way to solve. Command line parameter would be more suited.

**Comment by pawel-kow** [comment]
I think this PR does not fix #12, as this issue was more about enforcing an update even if IP check does not show any change, so not directly related to conflict detection.

Also in #11 the proposal was rather to have a way of using force parameter with `update` if conflicts are detected on the first time update, rather than always using force which might have unwanted side effects if the domain config would change and apply indeed should fail.

**Comment by pawel-kow** [comment]
Close due to lack of activity.

**Pull request closed without merging**